web/src/PickNeighbourhoodMode.svelte

```diff
@@ -214,24 +262,103 @@
   </div>
 
   <div slot="map">
-    <GeoJSON data={gj} generateId>
+    <GeoJSON data={neighbourhoods} generateId>
       <FillLayer
         {...layerId("neighbourhood-boundaries", false)}
         filter={["==", ["get", "kind"], "boundary"]}
         paint={{
           "fill-color": "black",
           "fill-opacity": hoverStateFilter(0.3, 0.5),
         }}
+        layout={{
+          visibility: selectedPrioritization == "none" ? "visible" : "none",
+        }}
         manageHoverState
+        bind:hovered={hoveredMapFeature}
+        hoverCursor="pointer"
         on:click={(e) =>
           pickNeighbourhood(notNull(e.detail.features[0].properties).name)}
+      >
+        <Popup openOn="hover" let:props>
+          <h2>{props.name}</h2>
+          <b>Area:</b>
+          {props.area_km2.toFixed(1)} km²
+        </Popup>
+      </FillLayer>
+      <FillLayer
+        {...layerId("neighbourhood-prioritization-simd")}
+        paint={{
+          "fill-color": makeRamp(["get", "simd"], simdLimits, simdColorScale),
+          "fill-opacity": hoverStateFilter(0.7, 0.9),
+        }}
+        layout={{
+          visibility: selectedPrioritization == "simd" ? "visible" : "none",
+        }}
+        manageHoverState
+        bind:hovered={hoveredMapFeature}
         hoverCursor="pointer"
+        on:click={(e) =>
+          pickNeighbourhood(notNull(e.detail.features[0].properties).name)}
       >
         <Popup openOn="hover" let:props>
-          <p>{props.name}</p>
+          <h2>{props.name}</h2>
+          <b>SIMD:</b>
+          {props.simd.toFixed(1)}
+        </Popup>
+      </FillLayer>
+      <FillLayer
```

Comment:
Hmm, now that I see how the multiple fill layers are duplicated here too, I think one Svelte component that's responsible for one FillLayer and takes in `selectedPrioritization` as a prop is sounding nicer. But we can leave as an idea for later.